spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+
+class Todo::Create
+  public :require
+end
```

Comment:
I ran into some issues with rails strong params (params.require() being a private method) when running the tests, so I used this patch to temporarily make the method public.

After a bit of reading on trailblazer, strong params can be done away with.